lib/fastlane/plugin/firebase_app_distribution/actions/login.rb

```diff
@@ -0,0 +1,65 @@
+require 'googleauth'
+require 'googleauth/stores/file_token_store'
+require "google/apis/people_v1"
+# require "googleauth"
+# require "googleauth/stores/file_token_store"
+require "fileutils"
+module Fastlane
+  module Actions
+    module SharedValues
+      TEST_AUTH_CUSTOM_VALUE = :TEST_AUTH_CUSTOM_VALUE
```

Comment:
It looks like this is unused. Can we remove this variable and/or the entire SharedValues module?